PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkLocalRunConfigurableModel.java

```diff
@@ -68,7 +75,16 @@
     public SparkLocalRunConfigurableModel(@NotNull Project project) {
         this.project = project;
         this.setWorkingDirectory(PathUtil.getLocalPath(project.getBaseDir().getPath()));
-        this.dataRootDirectory = "";
+        this.setDataRootDirectory(Paths.get(this.getWorkingDirectory(), "data").toString());
+
+        if (SystemUtils.IS_OS_WINDOWS) {
+            Optional.ofNullable(System.getenv(HADOOP_HOME_ENV))
+                    .map(hadoopHome -> Paths.get(hadoopHome, "bin", WINUTILS_EXE_NAME).toString())
+                    .map(File::new)
+                    .filter(File::exists)
+                    .map(winUtilsFile -> winUtilsFile.getParentFile().getParent())
+                    .ifPresent(hadoopHome -> this.envs.put(HADOOP_HOME_ENV, hadoopHome));
+        }
```

Comment:
I think maybe we can just put winutils.exe into our released package incase customers do not have the file.

Reply:
We can consider this idea. 😄